tap_spotify/streams.py

```diff
@@ -162,3 +162,14 @@ class GlobalViralTracksDailyStream(_PlaylistTracksStream):
     name = "global_viral_tracks_daily_stream"
     path = "/playlists/37i9dQZEVXbLiRSasKsNU9"
     primary_keys = ["rank", "synced_at"]
+
+
+class UserSavedTracksStream(SpotifyStream):
+    """Define global viral tracks daily stream."""
+
+    name = "user_saved_track_streams"
```

Comment:
```suggestion
    name = "user_saved_track_stream"
```

We want to keep the stream name format in line with other streams.